projects/plugins/jetpack/modules/videopress/class.videopress-edit-attachment.php

```diff
@@ -370,7 +375,8 @@ protected function display_rating( $info ) {
 			'R-17'  => 'R',
 		);
 
-		$displayed_rating = $info->rating;
+		$displayed_rating = isset( $info->rating ) ? $info->rating : 'G';
```

Comment:
I wonder whether defaulting to "G" really makes sense, versus leaving all the radio buttons unselected.

It looks like the rest of the code defaults to null, only applying a rating when one is supplied. Even the code reading this form submission already handles the case where none of the radio buttons is checked: https://github.com/Automattic/jetpack/blob/f74260c83d7d3acc491710ce97b95c69642c3f31/projects/plugins/jetpack/modules/videopress/class.videopress-edit-attachment.php#L90

Reply:
Done in 1f9a0271b3e3f08c1b773da2a9a80606cd020f83